framework/include/geomsearch/NearestNodeLocator.h

```diff
@@ -69,6 +69,12 @@ class NearestNodeLocator : public Restartable
   NodeIdRange & slaveNodeRange() { return *_slave_node_range; }
 
   /**
+   * Reconstructs the KDtree, updates the patch for the nodes in recheck_slave_nodes,
```

Comment:
This is a little detail, but it would probably be better to rename `recheck_slave_nodes` to simply `slave_nodes` both here and in the source for this function, since this might be used for purposes other than rechecking nodes.